docs/requirements.txt

```
@@ -5,3 +5,6 @@ myst-parser
 myst-nb
 sphinx-copybutton
 sphinx-autodoc-typehints
+fsspec
+rechunker
```

```suggestion
git+https://github.com/pangeo-data/rechunker.git
```

Ryan's PR has been merged into the dev branch, so using that should fix the API build until the changes are incorporated into the next stable release.

Sorry, didn't mean to delete the original review!